src/main/resources/application.properties

```diff
@@ -8,7 +8,12 @@ azure.activedirectory.client-secret=
 azure.activedirectory.user-group.allowed-groups=group1
 
 azure.keyvault.uri=https://mykeyvault1.vault.azure.net/
-
+# Specifies your Active Directory ID:
+azure.keyvault.tenant-id=
```

Comment:
Thanks Rujun for the updates.
The example I have is using Managed identity. Can you please check if the tenant/client id is required if Managed Identity is enabled?

Reply:
@milindvb Thank you for your review.

> Can you please check if the tenant/client id is required if Managed Identity is enabled?

No. not required if managed identity is enabled.

I created this PR because I received email which said client_credentials(client-id and client-secret) can not used to get access_token.

Reply:
FYI: After discuss with `Victor Colin Amador` in Teams, the problem has been solved.